config/default.js

```diff
@@ -136,4 +136,10 @@ module.exports = {
       missing: 'Workflow \'%workflow\' for Content Type \'%type\' not found',
     },
   },
+  storage: {
+    type: 'fs',
```

Comment:
actually, you've started it just by having that link! Can link to that list, then explain a bit more about how to drop in one of them?